File: JavaScript/ReverseWordsInAStringIII/ReverseWordsInAStringIII.js
/**
 * (202ms)
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    return s.split(' ').map(function(item) {return item.split('').reverse().join('')}).join(' ');
	// shorter
	// return s.split(' ').map(item => item.split('').reverse().join('')).join(' ')
};

File: Java/ReverseWordsInAStringIII/ReverseWordsInAStringIII.java
/**
 * (18ms)
 * @author Phoenix-Z
 *
 */
public class ReverseWordsInAStringIII {

	public String reverseWords(String s) {
        StringBuilder ans = new StringBuilder();
        for(String piece : s.split(" ")) {
            StringBuilder now = new StringBuilder(piece);
            ans.append(now.reverse()).append(' ');
        }
        return ans.deleteCharAt(ans.length() - 1).toString();
    }
	
}